essay.tex

```diff
@@ -11,35 +11,71 @@
 
 Your essay must make a clear recommendation, in terms of which of the three techniques you have reviewed is the best according to whichever metric or metrics you feel is most appropriate. You must justify your choice, backing it up with empirical evidence. However remember that an academic essay is not a murder mystery: you should already have briefly discussed your recommendation in the introduction and in other parts of the essay. Do not save it for a grand reveal at the end.
+\fi
+Developing digital games is hard and it gets even harder when you need to develop games with a team, so much more can go wrong when multiple people are working together on the same project, this is why a lot of people within the games industry practice Agile development methods; such as Scrum.
+\\~\\
+Scrum has an incremental development process known as Sprints, Sprints are pre-planned time periods in which the team has decided on what they will work on during the sprint. Sprints are a very important part of the Scrum methodology as they give a specific time limit for the team to work on tasks to implement into the project \cite{five} \cite{eight}.\\
+At the end of a Sprint, a sprint review meeting occurs in which the team gets together to discuss what happened during the sprint and update their project to the latest build of the game with all the new features and fixes that were made during the sprint\cite{eight}.\\
+Some developers like to use sprint reviews as a way of updating their fans on what they have been working on lately for the game, other developer who have early access games might even update their game after each sprint to give the fans regular new features and bug fixes in order to keep the game fresh for the players.
+\\~\\
+This is what the Agile manifesto says about sprints ``Deliver working software frequently, from a couple of weeks to a couple of months, with a preference to the shorter timescale.``\cite{three}.
+\\~\\
+So Sprints are an important part of the development cycle for games and they need to be well planned and organized in order for the team to work efficiently on a game.
+
+\section{So how long should a sprint last?}
+A lot of sources say that sprints should last 2-4 weeks depending on the team and the scale of the project \cite{two} \cite{four} \cite{five}. Some sources that have been explored also suggest that shorter sprints are generally better for software and games development \cite{three}. Shorter sprints generally work in favour for the Agile Scrum methodology for being short iterative steps towards development. A lot of teams have mentioned that they generally have 2 week long sprints as their main sprint duration as it gives a good time frame for developers to work on the scrum backlog. Any more than 4 weeks is usually considered too long and unfitting for the agile philosophy as it means the developers aren't iterating as much.
```

Comment:
Tighten up the style. Be concise. For example: `many sources suggest that sprints...` rather than `a lot of sources say`.